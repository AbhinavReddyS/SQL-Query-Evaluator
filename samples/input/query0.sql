SELECT * FROM Sailors, Boats where Sailors.A = 5 order by Boats.D;