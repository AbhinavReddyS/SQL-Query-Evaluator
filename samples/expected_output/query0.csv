5,100,500,101,2,3
5,100,500,102,3,4
5,100,500,103,1,1
5,100,500,104,104,2
5,100,500,107,2,8
